addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -473,29 +468,6 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
         }
         targetEl.classList.remove('active');
     },
-    /**
-     * Used to adjust the highlight effect when the text content is edited.
-     *
-     * @private
-     * @param {HTMLElement} target
-     * @param {String} highlightID
-     */
-    _adaptHighlightOnEdit(target, highlightID) {
```

Comment:
Let's keep the various refactoring this commit does in stable so we do not have different codebase that close to 17.0... but let's be a bit more careful: keep existing function, just stop using them and mark them to be removed in master.

Reply:
OK :+1: I did this for other functions, but for this one, I think it can be removed since [there is already a code to set it in start()](https://github.com/odoo/odoo/pull/142968/files#diff-d2188391a9d83cc97f3220e08d259e82796d94191dcf5e5fdb9b77e57074e6a5R88), no?

Reply:
Let's just not take the risk. Actually maybe just keep this line: `this._adaptHighlightOnEdit = throttleForAnimation(this._adaptHighlightOnEdit.bind(this));` and here in the definition of `_adaptHighlightOnEdit` just call `switchTextHighlight`, then in master you'll simplify during forward-port.

FYI, even if defined like that in the `start`, any custo can still break it... simply by calling it before start by mistake. It's wrong if someone did that... but their code would not crash and would start crashing after merging this PR :grimacing: